### PR TITLE
ENCM-197-ting-wang-support

### PR DESCRIPTION
**Pull request by khzlin**

### CODE REVIEW

File: src/encoded/schemas/analysis_step.json [hunk 1]
```diff
@@ -364,6 +367,8 @@
                     "transcriptome alignments",
                     "transcriptome index",
                     "transcriptome reference",
+                    "Transposable Element TF ancestral origin percent by motif",
```
Comment (gabdank) [inline src/encoded/schemas/analysis_step.json]: lower case "transposable element TF .."

File: src/encoded/schemas/analysis_step.json [hunk 2]
```diff
@@ -634,6 +642,8 @@
...
```
Comment (gabdank) [inline src/encoded/schemas/analysis_step.json]: lower case "transposable element TF ..."

File: src/encoded/schemas/changelogs/analysis_step.md [hunk 3]
```diff
@@ -3,6 +3,8 @@
 ### Minor changes since schema version 17
 * Added *topic gene weights* and *cell topic participation* to enum list for *input_file_types* and *output_file_types*.
 * Added *topic modeling*, *TF binding site identification*, and *element annotation* to enum list for *analysis_step_types*.
+* Added *motif clusters reference*, *phastcons score reference*, *Transposable Element TF ancestral origin percent by motif*, *Transposable Element TF ancestral origin percent by subfamily* and *variant reference* to enum list for *input_file_types* and *output_file_types*.
```
Comment (gabdank) [inline src/encoded/schemas/changelogs/analysis_step.md]: lower case "transposable.."

File: src/encoded/schemas/changelogs/file.md [hunk 4]
```diff
@@ -3,6 +3,7 @@
 ### Minor changes since schema version 31
 * Added *topic gene weights* and *cell topic participation* to enum list for *output_type*.
 * Added *yaml* to enum list for *file_format*.
+* Added *motif clusters reference*, *phastcons score reference*, *Transposable Element TF ancestral origin percent by motif*, *Transposable Element TF ancestral origin percent by subfamily*, *repeats reference*, and *variant reference* to enum list for *output_type*.
```
Comment (gabdank) [inline src/encoded/schemas/changelogs/file.md]: lower case

File: src/encoded/schemas/file.json [hunk 5]
```diff
@@ -2358,6 +2361,8 @@
                         "transcriptome annotations",
                         "transcriptome index",
                         "transcriptome reference",
+                        "Transposable Element TF ancestral origin percent by motif",
```
Comment (gabdank) [inline src/encoded/schemas/file.json]: lower case

File: src/encoded/schemas/file.json [hunk 6]
```diff
@@ -3063,6 +3069,8 @@
         "enhancer-gene links": "annotation",
         "thresholded links": "annotation",
         "thresholded element gene links": "annotation",
+        "Transposable Element TF ancestral origin percent by motif": "annotation",
```
Comment (gabdank) [inline src/encoded/schemas/file.json]: lower case

File: src/encoded/tests/data/inserts/analysis_step.json [hunk 7]
```diff
@@ -726,12 +726,22 @@
...
+                "Transposable Element TF ancestral origin percent by subfamily",
+                "Transposable Element TF ancestral origin percent by motif",
+                "phastcons score reference",
+                "motif clusters reference",
+                "variant reference"
         ],
         "output_file_types": [
-                "fragments"
+                "fragments",
+                "Transposable Element TF ancestral origin percent by subfamily",
```
Comment (gabdank) [inline src/encoded/tests/data/inserts/analysis_step.json]: lower case

File: src/encoded/tests/data/inserts/expected_annotation_metadata.tsv [hunk 8]
```diff
@@ -1,11 +1,13 @@
...
 ENCFF129CVS	bigWig	candidate Cis-Regulatory Elements		ENCSR129CCR	candidate Cis-Regulatory Elements		current, ENCODE v2							Homo sapiens		2020-05-20	ENCODE	John Stamatoyannopoulos, UW	5c78ceaa13a380990ca16d8dcaeffd0b		http://localhost/files/ENCFF129CVS/@@download/ENCFF129CVS.bigWig	hg19		released	/files/ENCFF002COS/, /files/ENCFF129CRE/, /files/ENCFF129DNA/			200	inconsistent assembly, missing analysis_step_run		
+ENCFF001KEK	tsv	Transposable Element TF ancestral origin percent by subfamily		ENCSR767PSZ	transcription factor ancestral origin estimate									Homo sapiens		2023-03-15	ENCODE	Michael Snyder, Stanford	a94628feb64c6c2050181e45e3d29fe4		http://localhost/files/ENCFF001KEK/@@download/ENCFF001KEK.tsv	hg19		released				289435	missing analysis_step_run		
+ENCFF002KEK	tsv	Transposable Element TF ancestral origin percent by motif		ENCSR767PSZ	transcription factor ancestral origin estimate									Homo sapiens		2023-03-15	ENCODE	Michael Snyder, Stanford	4f637c4a2dd0fccd6eed3d6ede385d35		http://localhost/files/ENCFF002KEK/@@download/ENCFF002KEK.tsv	hg19		released				289435	missing analysis_step_run
```
Comment (gabdank) [inline src/encoded/tests/data/inserts/expected_annotation_metadata.tsv]: lower case

File: src/encoded/tests/data/inserts/file.json [hunk 9]
```diff
@@ -6246,5 +6245,96 @@
...
+        "accession": "ENCFF002KEK",
+        "assembly": "hg19",
+        "award": "/awards/U54HG006996/",
+        "dataset": "ENCSR767PSZ",
+        "file_format": "tsv",
+        "date_created": "2014-07-09T16:46:18.670068+00:00",
+        "file_size": 289435,
+        "lab": "/labs/michael-snyder/",
+        "md5sum": "4f637c4a2dd0fccd6eed3d6ede385d35",
+        "output_type": "Transposable Element TF ancestral origin percent by motif",
```
Comment (gabdank) [inline src/encoded/tests/data/inserts/file.json]: lower case

File: src/encoded/tests/data/inserts/file.json [hunk 10]
```diff
@@ -6246,5 +6245,96 @@
...
+        "accession": "ENCFF001KEK",
+        "assembly": "hg19",
+        "award": "/awards/U54HG006996/",
+        "dataset": "ENCSR767PSZ",
+        "file_format": "tsv",
+        "date_created": "2014-07-09T16:46:18.670068+00:00",
+        "file_size": 289435,
+        "lab": "/labs/michael-snyder/",
+        "md5sum": "a94628feb64c6c2050181e45e3d29fe4",
+        "output_type": "Transposable Element TF ancestral origin percent by subfamily",
```
Comment (gabdank) [inline src/encoded/tests/data/inserts/file.json]: lower case

**Review by gabdank — CHANGES_REQUESTED** [review]
lower case for transposable elements

File: src/encoded/schemas/analysis_step.json (hunk 1 above)
Comment (khzlin) [reply src/encoded/schemas/analysis_step.json]: done

File: src/encoded/schemas/analysis_step.json (hunk 2 above)
Comment (khzlin) [reply src/encoded/schemas/analysis_step.json]: done

File: src/encoded/schemas/changelogs/analysis_step.md (hunk 3 above)
Comment (khzlin) [reply src/encoded/schemas/changelogs/analysis_step.md]: done

File: src/encoded/schemas/file.json (hunk 5 above)
Comment (khzlin) [reply src/encoded/schemas/file.json]: done

File: src/encoded/schemas/file.json (hunk 6 above)
Comment (khzlin) [reply src/encoded/schemas/file.json]: done

File: src/encoded/tests/data/inserts/expected_annotation_metadata.tsv (hunk 8 above)
Comment (khzlin) [reply src/encoded/tests/data/inserts/expected_annotation_metadata.tsv]: done

File: src/encoded/tests/data/inserts/file.json (hunk 10 above)
Comment (khzlin) [reply src/encoded/tests/data/inserts/file.json]: done

File: src/encoded/schemas/changelogs/file.md (hunk 4 above)
Comment (khzlin) [reply src/encoded/schemas/changelogs/file.md]: done

File: src/encoded/tests/data/inserts/file.json (hunk 9 above)
Comment (khzlin) [reply src/encoded/tests/data/inserts/file.json]: done

**Review by gabdank — DISMISSED** [review]

**Review by gabdank — DISMISSED** [review]

**Review by gabdank — APPROVED** [review]

**Pull request merged**